api/filter/http/gzip.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+// [#proto-status: experimental]
+
+package envoy.api.v2.filter.http;
+
+import "validate/validate.proto";
+
+// This is an HTTP filter which enables Envoy to compress dispatched data from an upstream
+// service upon client request. This is useful in situations where large payloads needs to
```

Comment:
typo here: "where a large payload needs to be" or "where large payloads need to be" both seem OK